### DISCUSSION TITLE
根目录下的config.py设置没有作用怎么解决？

### DISCUSSION THREAD

**Discussion by zimarchangel** [nonebot/discussions#50] [问答] [answered]
在bot.py同一个目录下建立的config.py，应该是有被import，但是config.py里面的设置都没有作用
按着https://docs.nonebot.dev/guide/basic-configuration.html#基本配置  里面的说明一步步做的
python版本是3.9.5
求助了，谢谢

**Comment by yanyongyu** [comment, marked as answer]
```python
import config
nonebot.init(config)
```

**Reply by zimarchangel** [reply]
就是这样处理的，但是没有用，表现在没有办法识别命令起始符
COMMAND_START: {'.'}
DEBUG: Parsing command: '.help'
DEBUG: It's not a command

**Reply by yanyongyu** [reply]
```diff
- COMMAND_START: {'.'}
+ COMMAND_START = {'.'}
```

**Reply by zimarchangel** [reply]
啊！谢谢！解决了！
（被自己蠢到）
